Product/product.css

```diff
@@ -0,0 +1,30 @@
+.product-filter {
+  width: 18.75rem;
+}
+
+.product-filter h4 {
```

Comment:
maybe use css combinators it will increase specificity and if possible try to use classes instead of elements.